website/content/posts/introducing-action-validate-image.md

```diff
@@ -0,0 +1,119 @@
+---
+title: "Introducing the latest addition Action Validate"
```

Comment:
Needs a colon I think:

```suggestion
title: "Introducing the latest addition: Action Validate"
```

..but I'd probably go with something shorter that mentions GitHub

```suggestion
title: "Introducing Action Validate for GitHub"
```